haystack/document_store/elasticsearch.py

```diff
@@ -60,6 +61,7 @@ def __init__(
         :param password: password (standard authentication via http_auth)
         :param api_key_id: ID of the API key (altenative authentication mode to the above http_auth)
         :param api_key: Secret value of the API key (altenative authentication mode to the above http_auth)
+        :param aws4auth: Authentication for usage with aws elasticsearch (can be generated with the requests-aws4auth package)
```

Comment:
I like that we do not add requests-aws4auth as dependency in our requirements.txt for now.

In your next PR you can directly write that you can install the lib with `pip install requests-aws4auth` to make it even easier for the users : )